src/http/server/response.cr

```diff
@@ -170,6 +170,28 @@ class HTTP::Server
       respond_with_status(HTTP::Status.new(status), message)
     end
 
+    # Sends a redirect to *location*.
+    #
+    # The value of *location* gets encoded with `URI.encode`.
+    #
+    # The *status* determines the HTTP status code which can be
+    # `HTTP::Status::FOUND` (`302`) for a temporary redirect or
+    # `HTTP::Status::MOVED_PERMANENTLY` (`301`) for a permanent redirect.
+    #
+    # The response gets closed.
+    #
+    # Raises `IO::Error` if the response is closed or headers were already
+    # sent.
+    def redirect(location : String | URI | Path, status : HTTP::Status = :found)
```

Comment:
I don't understand why we need to support `Path` here, especially when this type also includes Windows paths.

Reply:
Hm, that's a good point. I actually don't remember the reasoning. Athena allow `Path` as well, though.
But I think we can remove this type. If there is demand for it later, we can always reconsider.

Reply:
IIRC this was to support the case of a redirect to a directory/file. E.g. for a filesystem agnostic static file handler.